src/components/lead/styles/vars/CdrLead.vars.scss

```diff
@@ -3,14 +3,14 @@
   line-height: 0;
   &--top {
     background: linear-gradient(
-      var(--cdr-lead-base-background-color, var(--cdr-color-background-primary, #{$cdr-color-background-primary})) 0 50%,
+      var(--cdr-lead-surface-color, var(--cdr-color-background-primary, #{$cdr-color-background-primary})) 0 50%,
```

Comment:
I wonder if there is time to get 
--cdr-surface
--cdr-surface-recessed 
into tokens and then default the surface color to --cdr-surface - and allow the prop to switch its token?

Reply:
Can follow up with that, I want to get started on a round of integration testing